src/styles/pages/committees/index.module.scss

```diff
@@ -0,0 +1,39 @@
+
+
+.committees_page {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: var(--color-white);
+    
+    h1 {
+        padding-top: 46px;
+        padding-bottom: 26px;
+        color:var(--color-maroon);
+        font-family: "Be Vietnam";
+        font-size: 3.4375rem;
+        font-style: normal;
+        font-weight: 700;
+    }
+}
+
+
+
+.committees_cards {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    padding-left: 170px;
+    padding-right: 170px;
+}
+
+.committees_cards > div {
+    flex: 0 0 33%;
+    display: flex;
+    justify-content: center;
+    padding-bottom: 64.85px;
```

Comment:
you can round your px values to the nearest multiple of 8, don't need to be too specific.